partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/MultiPartnerServiceImpl.java

```diff
@@ -167,34 +163,34 @@ public List<PolicyDto> getAllPolicies() {
     }
 
     @Override
-    public List<PartnerTypesDto> getDetailsForAllPartnerTypes() {
+    public List<PartnerTypesDto> getDetailsForAllPartnerTypes(String status) {
         List<PartnerTypesDto> partnerTypesDtoList = new ArrayList<>();
         try {
             String userId = getUserId();
             List<Partner> partnerList = partnerRepository.findByUserId(userId);
             if (!partnerList.isEmpty()) {
                 for (Partner partner : partnerList) {
-                    PartnerTypesDto partnerTypesDto = new PartnerTypesDto();
-                    try {
-                        if (Objects.isNull(partner.getId()) || partner.getId().equals(BLANK_STRING)) {
-                            LOGGER.info("Partner Id is null or empty for user id : " + userId);
-                            throw new PartnerServiceException(ErrorCode.PARTNER_ID_NOT_EXISTS.getErrorCode(),
-                                    ErrorCode.PARTNER_ID_NOT_EXISTS.getErrorMessage());
-                        }
-                        String policyGroupName = policyGroupRepository.findPolicyGroupNameById(partner.getPolicyGroupId());
-                        if (Objects.isNull(policyGroupName) || policyGroupName.equals(BLANK_STRING)) {
-                            LOGGER.info("Policy Group Name is null or empty for partner id : " + partner.getId());
-                            throw new PartnerServiceException(ErrorCode.POLICY_GROUP_NOT_EXISTS.getErrorCode(),
-                                    ErrorCode.POLICY_GROUP_NOT_EXISTS.getErrorMessage());
+                    String partnerType = partner.getPartnerTypeCode();
+                    if (!partnerType.equalsIgnoreCase("Device_Provider") && !partnerType.equalsIgnoreCase("FTM_Provider")
```

Comment:
there must be some common constants for partner types, use those instead of adding strings

Reply:
defined common constants for partnerTypes